.circleci/config.yml

```diff
@@ -71,6 +79,10 @@ jobs:
           name: Ensure database is available
           command: dockerize -wait tcp://localhost:5432 -timeout 1m
 
+      - run:
+          name: Create database roles
+          command: |
+              psql $SQLA_TEST_CONN -f data/create_roles.sql
```

Comment:
As noted in our overview conversation, we'll probably want to create an admin role specific for flyway for ATO/audit purposes.

/cc @LindsayYoung:  other than creating the role in the RDS environments and documenting that, is there anything else we need to do that I'm forgetting?

Reply:
Agreed.

Reply:
Caught up with @LindsayYoung about this, we'll want to either leverage the existing API account we have created in RDS, or create a new one just for this.